latex/Specific Aims.tex

```diff
@@ -124,21 +124,25 @@
 % Aim 4: SAMPL -- Goal is learning from the above (a) where we are now, (b) why we fail/what we're missing, and (c) how to do better
 %I've tried to encapsulate that in these revised aims.
 
-{\bf \underline{Aim 1.} Generate new data for ``simple'' SAMPL blind challenges on physical property prediction}\\
+%{\bf \underline{Aim 1.} Generate new data for ``simple'' SAMPL blind challenges on physical property prediction}\\
+{\bf \underline{Aim 1.} Spur improvements in modeling protonation states, tautomers, and solvation via generation of new physical property datasets.}\\
```

Comment:
How about this: "Collect new physical property datasets to assess accuracy and spur improvements in force fields and modeling of protonation states and tautomers."

Elements of yours and mine...